### ISSUE TITLE
Create backup from server data

### ISSUE THREAD

**Issue by MR-Os1** [traggo/server#143]
Is there a way to properly backup server's data to a file and restore it?
if it's not, how can we add it?

**Comment by jmattheis** [comment]
Yes, to back up all traggo data, you can copy the traggo database. All state is in there.
https://github.com/traggo/server/blob/b707a27fecb60b454d4421222b8eecc5bb184161/.env.sample#L17-L19

**Comment by oslavq** [comment]
Traggo stores all of it's data in a `.db` (database) file, so backing it up yourself is pretty easy.

cd into traggo data location

make sure you got at least `rx` permissions for `traggo.db` and `sqlite3` package installed on your system. Use these commands to create a [database dump](https://en.wikipedia.org/wiki/Database_dump) (it will contain all of the data from the database as SQL queries that can be later used to recreate the database)

`cd /opt/traggo/data` (for you, the folder might be in a different location)
`sqlite3 traggo.db`
`.output dump_name.sql`
`.dump`
exit the sqlite3 shell with `Ctrl + C`

now you will have a .sql file with all of your traggo data. You can view it as plain text with any text editor

You can find guides online on how to restore from a dump, so i won't write this down here.
